File: C/section06/S06E10.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int swimmerAge;

    printf("Please enter the swimmer's age. \n");
    scanf("%d", &swimmerAge);

    if (swimmerAge >= 18) {
        printf("\nThe swimmer is at adult category.\n");
    } else if (swimmerAge >= 14) {
        printf("\nThe swimmer is at juvenile-b category.\n");
    } else if (swimmerAge >= 12) {
        printf("\nThe swimmer is at juvenile-a category.\n");
    } else if (swimmerAge >= 8) {
        printf("\nThe swimmer is at child-b category.\n");
    } else {
        printf("\nThe swimmer is at child-a category.\n");
    }  

    return 0;
}

File: Python/section06/S06e10.py
swimmerAge = int(input ("Please enter the swimmer age: "))

if swimmerAge >= 18:
    print("Adult.")
elif swimmerAge >= 14:
    print("juvenile B.")
elif swimmerAge >= 12:
    print("juvenile A.")
elif swimmerAge >= 8:
    print("child B.")
elif swimmerAge >= 5:
    print("child A.")
else:
    print("This swimmer is not fit to any category.")